guides/updating/versions/0.3-to-0.4.md

```diff
@@ -0,0 +1,875 @@
+
+The most common use case for the `publicDid` property was for registering an endorser did that can endorse (read: pay for) transactions on the ledger. This can now be done by importing the did into agent, aftre which it can be used by the anoncreds module to register schemas and credential definitions, and the did registrar to register dids.
+
+There's a **one-time import** that needs to be done to import the did into the agent using the `DidsApi.import` method. If you've previously used the `publicDidSeed` property, providing the private key is optional, as it is already stored in the wallet. Note that the `privateKey` is the same as the `publicDidSeed`, see [Changes to Wallet](#changes-to-wallet) for context). The `import` method will resolve the did passed to the `import` method (so make sure to regiser the `IndySdkIndyDidResolver`). It is recommended to import the endorser did as an `did:indy` did rather than a `did:sov` did, as the `did:sov` method is deprecated for creation. The `did:indy` did can be constructed by replacing `sov` with `indy:<indyNamespace>`, where `<indyNamespace>` should be replaced with a namespace as registered in the `networks` property of the `IndySdkModule` (see [Indy Network Configuration](#indy-network-configuration)).
+
+An inital list of namespace identifiers can be found in the [Indy did resolver driver](https://github.com/decentralized-identity/uni-resolver-driver-did-indy/blob/main/docker/Dockerfile#LL39C6-L39C6) for the universal resolver project, in the future this list will be maintained in the [Indy DID Networks repository](https://github.com/hyperledger/indy-did-networks).
+
+<!--tabs-->
+
+##### 0.3.x
+
+```ts
+import { Agent } from '@aries-framework/core'
+
+// Import from @aries-framework/react-native in React Native
+import { agentDependencies } from '@aries-framework/node'
+
+const agent = new Agent({
+  config: {
+    publicDidSeed: '01eafa4de4e22ed4fc2ee522b6ce2731',
+  },
+  dependencies: agentDependencies,
+})
+```
+
+##### 0.4.x
+
+```ts
+import { Agent, DidsModule, KeyType, TypedArrayEncoder } from '@aries-framework/core'
+import { IndySdkModule, IndySdkIndyDidResolver } from '@aries-framework/indy-sdk'
+
+// Import from @aries-framework/react-native in React Native
+import { agentDependencies } from '@aries-framework/node'
+
+// Import from indy-sdk-react-native in React Native
+import indySdk from 'indy-sdk'
+
+const agent = new Agent({
+  config: {
+    /* ... */
+  },
+  dependencies: agentDependencies,
+  modules: {
+    indySdk: new IndySdkModule({
+      indySdk,
+      networks: [
+        {
+          id: 'Sovrin Mainnet',
+          // Important: make sure to pick the correct indy namespace for the network you're connecting to
```

Comment:
Do we include a link somewhere on where to find it? Maybe the anoncreds registry list? Not sure if there are better resources for this.

Reply:
There's not really a list. I'll open an issue in indy-did-networks and point towards it